dot/parachain/collator-protocol/collation_fetching.go

```diff
@@ -1,16 +1,24 @@
 // Copyright 2023 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
-package parachain
+package collatorprotocol
 
 import (
 	"fmt"
+	"time"
 
 	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 )
 
+const (
+	/// Maximum PoV size we support right now.
+	maxPoVSize                       = 5 * 1024 * 1024
```

Comment:
```suggestion
	maxPoVSize                       = 6 * 1024
```

Reply:
looks like you might be confusing last * with a +

Reply:
ooh, yeah, nervermind!